### PR TITLE
adds the knockback stick

### PR DESCRIPTION
**Pull request by qwertyquerty**
because why not

## Changelog
:cl:
add: Added the knockback stick
/:cl:

### CODE REVIEW

File: code/game/objects/items/melee/misc.dm [hunk 1]
```diff
@@ -469,3 +469,42 @@
...
+
+	if(cooldown <= world.time)
+		playsound(get_turf(src), 'sound/effects/woodhit.ogg', 75, 1, -1)
+		log_combat(user, target, "knockedbacked", src)
+		target.visible_message("<span class ='danger'>[user] has knocked back [target] with [src]!</span>", \
+			"<span class ='userdanger'>[user] has knocked back [target] with [src]!</span>")
+
+		var/throw_dir = get_dir(user,target)
+		var/turf/throw_at = get_ranged_target_turf(target, throw_dir, knockbackpower)
+		target.throw_at(throw_at, throw_range, 3)
```
Comment (ike709) [inline code/game/objects/items/melee/misc.dm:503]: `target.safe_throw_at(`

Comment (qwertyquerty) [reply code/game/objects/items/melee/misc.dm:503]: Why

**Comment by qwertyquerty** [comment]
Checks failed due to it not being able to download mariadb or something weird

File: code/game/objects/items/melee/misc.dm (hunk 1 above)
Comment (qwertyquerty) [reply code/game/objects/items/melee/misc.dm:503]: I don't want that. It's meant to be a gimmick item not something players just get without an admin spawn. It shouldn't hold back

**Pull request merged**